MAUI/Barcode/customization.md

```diff
@@ -0,0 +1,364 @@
+        barcode.Symbology = new Codabar();
+        this.Content = barcode;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![.NET MAUI Barcode Generator Barcode Without Module](images/customization/maui-barcode-without-module.png)
+
+N>  In the image above, the smallest bar line width is calculated on the basis of the available width as a result of the total computed inputs(0’s and 1’s) divided by the available width.
+
+For two dimensional barcode , if the `Module` property is not set, the size of smallest dot is calculated based on the minimum of available width or height.
+
+The following code snippet shows the two dimensional barcode with `Module` property,
+
+{% tabs %}
+
+{% highlight xaml hl_lines="6" %}
+
+            <barcode:SfBarcodeGenerator Value="123456789" 
+                                        HeightRequest="150"
+                                        WidthRequest="230"  
+                                        BackgroundColor="LightCyan">
+                <barcode:SfBarcodeGenerator.Symbology>
+                    <barcode:QRCode Module="2"/>
+                </barcode:SfBarcodeGenerator.Symbology>
+            </barcode:SfBarcodeGenerator>
+
+{% endhighlight %}
+
+{% highlight C# hl_lines="6" %}
+
+        SfBarcodeGenerator barcode = new SfBarcodeGenerator();
+        barcode.HeightRequest = 150;
+        barcode.WidthRequest = 230;
+        barcode.Value = "123456789";
+        barcode.BackgroundColor = Colors.LightCyan;
+        barcode.Symbology = new QRCode() { Module = 2 };
+        this.Content = barcode;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![.NET MAUI Barcode Generator QR Code Width Customization](images/customization/maui-qrcode-width-customization.png)
+
+Below code snippet shows the two dimensional barcode without the `Module` property,
+
+{% tabs %}
+
+{% highlight xaml %}
+
+            <barcode:SfBarcodeGenerator Value="123456789" 
+                                        HeightRequest="150"
+                                        WidthRequest="230"  
+                                        BackgroundColor="LightCyan">
+                <barcode:SfBarcodeGenerator.Symbology>
+                    <barcode:QRCode />
+                </barcode:SfBarcodeGenerator.Symbology>
+            </barcode:SfBarcodeGenerator>
+
+{% endhighlight %}
+
+{% highlight C# %}
+
+        SfBarcodeGenerator barcode = new SfBarcodeGenerator();
+        barcode.HeightRequest = 150;
+        barcode.WidthRequest = 230;
+        barcode.Value = "123456789";
+        barcode.BackgroundColor = Colors.LightCyan;
+        barcode.Symbology = new QRCode();
+        this.Content = barcode;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![.NET MAUI Barcode Generator QR Code Without Module](images/customization/maui-qrcode-without-module.png)
+
+### Bar color customization
+
+The bar color of the barcode can be customized by using its `ForegroundColor` property as like the following code snippet,
```

Comment:
The bar color of the barcode can be customized by using the `ForegroundColor` property as shown in the following code sample.

Reply:
Changes updated.